qute.ls/com.redhat.qute.ls/src/main/java/com/redhat/qute/project/tags/UserTag.java

```diff
@@ -115,16 +115,38 @@ private List<String> createBody() {
 	 */
 	public static void generateUserTagParameter(UserTagParameter parameter, boolean snippetsSupported, int index,
 			StringBuilder snippet) {
+		// Generate parameter name
 		snippet.append(parameter.getName());
 		snippet.append("=");
-		snippet.append("\"");
+
+		// Generate parameter value
+		String value = parameter.getName();
+		char quote = '"';
+		String defaultValue = parameter.getDefaultValue();
+		if (defaultValue != null && !defaultValue.isEmpty()) {
+			value = defaultValue;
+			// there is a default value, remove the quote if needed
+			char start = defaultValue.charAt(0);
+			if (start == '"' || start == '\'') {
+				quote = start;
+				value = value.substring(1, value.length() - (value.endsWith(start + "") ? 1 : 0));
+			} else {
+				quote = (char) -1;

```

Comment:
> This uses the character of -1

So bad, I fixed it by using Character / null

Reply:
> In order to get this completion, you need the following Qute template (I think it's invalid since the quotes are missing):

I don't know, please create a new issue for that if we needto report an error in this case.

Reply:
I need to avoid generating quote in case you have a parameter like `name=10`, it i sjust a copy/paste of the 10 value.

In case of quote, I need to so some extra work to generate the quote before the snippet syntax and not inside, in order to after apply of completion, the plcaholder which must be selected is the value and not the "value" (with quote)

Reply:
You change fixed the completion. qute-ls already reports `{#let punctuation=.}` as an error, so I won't file an issue for that.

Reply:
Ok so we are good